### ISSUE TITLE
Manual URL's in TinyMCE are broken when using multiple languages

### ISSUE THREAD

**Issue by MrTango** [plone/Products.CMFPlone#3633]
### What I did:

when links are inserted direktly as HTL or by using the external link field of Tinymce, the links are broken when more then one language is active. Because the links are missing the language folders.

### What version of Plone/ Addons I am using:

Plone 5.2

**Comment by thet** [comment]
@MrTango I don't think that https://github.com/plone/Products.CMFPlone/pull/3765 will address this problem.
Can you give more details on this error? Are there any console errors?

**Comment by thet** [comment]
That's then a  problem with `ResolveUIDAndCAptionFilter` in `plone.outputfilters` AFAIK.

**Comment by MrTango** [comment]
No it is the tinyMCE who changes this to a relative link.
You can check by looking at the html source, after inserting the link.
It will no longer have the full url but the relative link.

**Comment by petschki** [comment]
I think this is fixed by https://github.com/plone/mockup/issues/1116

**Comment by jensens** [comment]
Closing as resolved. Fixed: mockup#1116 resolved the multilingual TinyMCE URL issue.
Thank you for the original report!
